spec/user_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe User, type: :model do
```

Comment:
It's a good practice to test all features. But here in this card only tests on Books model are asked.